src/tracing.rs

```diff
@@ -1,3 +1,11 @@
+use std::{
+    fmt,
+    sync::{Arc, Mutex, Once},
+};
+
+use ::tracing::Level;
```

Comment:
The leading `::` should not be needed, since there is no local symbol called `tracing` in scope.

Reply:
hrm, my IDE seemed to want to use the current file. Can try removing it.